src/NServiceBus.Core/Licensing/ConfigureLicenseExtensions.cs

```diff
@@ -32,14 +31,8 @@ public static void LicensePath(this EndpointConfiguration config, string license
         {
             Guard.AgainstNull(nameof(config), config);
             Guard.AgainstNullAndEmpty(nameof(licenseFile), licenseFile);
-            if (!File.Exists(licenseFile))
-            {
-                throw new FileNotFoundException("License file not found", licenseFile);
-            }
-
-            var licenseText = NonLockingFileReader.ReadAllTextWithoutLocking(licenseFile);
-
-            config.License(licenseText);
+            Logger.Info("Using license supplied via fluent API.");
```

Comment:
should we indicate that a *license file path* has been provided via fluent API (instead of just *license*) and maybe also print out the path?